<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frontControlPanel</class>
 <widget class="QWidget" name="frontControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>411</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Error</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutCode">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserErrorCode"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutDescription">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserErrorDescription"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutPart">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Part</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserErrorPart"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonErrorReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButtonInitialize">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Initialize</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPause">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
